extmod/modtime.c

```diff
@@ -134,6 +134,8 @@ static mp_obj_t time_sleep_ms(mp_obj_t arg) {
     mp_int_t ms = mp_obj_get_int(arg);
     if (ms >= 0) {
         mp_hal_delay_ms(ms);
+    } else {
+        mp_handle_pending(true);
```

Comment:
Hmm, I'm not sure this bit needs to be added.  There doesn't seem to be anything broken that needs this change.

Reply:
I don't understand.

The existing logic here is `if (ms >= 0)`, so this change won't change anything for `time.sleep_ms(0)`.